src/components/Catalog/Catalog.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/utils/vars.scss';
+@import '../../styles/utils/mixins.scss';
+
+.catalogContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  margin: 0 auto;
+
+  max-width: 1168px;
+
+  @include onDesctop {
+    width: calc(3 * #{$productCard-width} + 2 * #{$productCards-gap-column});
```

Comment:
Why do we need fixed catalog width? Seems better to just place it in the center and let them have all the width.